clang/lib/CIR/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+include_directories(${LLVM_MAIN_SRC_DIR}/../mlir/include)
+include_directories(${CMAKE_BINARY_DIR}/tools/mlir/include)
```

Comment:
I think something like `target_include_directories` is generally preferred, because it is less of a sledgehammer effecting many targets at once.